pkg/META

```diff
@@ -1,5 +1,5 @@
 description = "POSIX time for OCaml"
-version = "%%VERSION_NUM%%"
```

Comment:
This file is not used by the dune port is it?

Reply:
That's correct. This change was made by running the `ocaml pkg/pkg.ml build --dev-pkg true` command. Since the port does require some of the output of that command to be checked in (e.g. the generated code) I figured it would be simplest to just check in all of its output.